package.json

```diff
@@ -25,6 +25,7 @@
     "supertest": "^6.3.0"
   },
   "dependencies": {
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "uuid": "^9.0.0"
```

Comment:
Make sure to avoid adding unneeded deps. `uuid` is not needed, as requirement calls for integer ids.
```suggestion
```